docs/extensions/info-xml.md

```diff
@@ -250,15 +250,15 @@ Elements acceptable within `<maintainer>`
 ### `<psr4>` {:#psr4}
 
 * Containing element: [`<classloader>`](#classloader)
-* Description: *not yet documented*
+* Description: Specifies the Name space and patho of classes to add the Composer Classloader
```

Comment:
Typos. Inconsistent caps. We extend composer's class-loader, but that's an implementation detail -- all that matters here is that it's PSR-4 compliant.

Maybe: "Specifies the namespace and file-path of classes to add the classloader."